adk/mlops.py

```diff
@@ -0,0 +1,44 @@
+import yaml
+import json
+import os
+import subprocess
+
+
+class MLOps(object):
+    spool_dir = "/tmp/ta"
+    agent_dir = "/opt/mlops-agent/datarobot_mlops_package-8.1.2"
```

Comment:
I'd think of this this version number like a dependency and consider not hard-coding , just like you wouldn't for other libraries, as it's going to change relatively frequently. Maybe put in the config file that you read in?

Reply:
@zeryx For flexibility and less configuration, how about just reading the insides of `opt/mlops-agent` directory and getting the dir that contains `datarobots_mlops_package` in its name?

Reply:
However, that hardcoded version number is coming from `libraries/mlops-agent/Dockerfile` from the langpacks (see https://github.com/algorithmiaio/langpacks/pull/228) 

So this means that whenever we'd like to update the mlops agent version, we'd update the langpacks Dockerfile anyway, unless there's a more flexible approach to break this dependency.

Reply:
Yeah I think any dynamic reading approach would probably be preferable over hardcoded. It looks like you're on the right track; not sure what the exact best option is, and may still involve updating multiple places, but at least extract it from the code itself I think.